modules/cudf/src/data_frame.ts

```diff
@@ -223,6 +224,16 @@ export class DataFrame<T extends TypeMap = any> {
   /** @ignore */
   asTable() { return new Table({columns: this._accessor.columns}); }
 
+  /**
+   *
+   * @param options
+   * @returns void
+   */
+  toString(options?: DisplayOptions): string {
```

Comment:
Could do

`toString(options: DisplayOptions = {}): string {`

and then avoid the `??`, but it's up to what you prefer.

Reply:
https://github.com/rapidsai/node/pull/297/commits/6853b924285a69689aa481378e551aa295b4a947